config/config.js

```diff
@@ -88,6 +88,28 @@ var validateSecureMode = function (config) {
 };
 
 /**
+ * Validate Session Secret parameter is not set to default in production
+ */
+var validateSessionSecret = function (config, testing) {
```

Comment:
@lirantal why is an anonymous function assigned to a variable better than a named function?